### PR TITLE
Update the Jail Wizard tooltips

### PR DESCRIPTION
**Pull request by miker**
Updated the tooltips to the new formatting style
The Jail Wizard is for simple jail configuration.

The IPv4 and IPv6 address fields only allow a single IP to be entered.
(The regex validation confirms) Updated the tooltips to reflect this.

After linking to specific sections of the User Guide works, there
may be other updates/clarifications that can be made.

Tooltips in this update are geared towards Wizard and newbie use.

### CODE REVIEW

**Comment by erinix** [comment]
could you please fix your merge conflicts

File: src/app/pages/jails/jail-wizard/jail-wizard.component.ts [hunk 1]
```diff
@@ -54,17 +56,19 @@ export class JailWizardComponent {
...
           type: 'checkbox',
           name: 'vnet',
           placeholder: T('VirtIO Virtual Networking'),
-          tooltip: T('Check to use VirtIO to emulate network devices for\
- this jail. See <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=virtio&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">VIRTIO(4)</a> for more details.'),
+          tooltip: T('Use VirtIO to emulate network devices for the \
+                      jail. <br> \
+                      See <a href="https://www.freebsd.org/cgi/man.cgi?\
```
Comment (Mrt134) [inline src/app/pages/jails/jail-wizard/jail-wizard.component.ts]: Don't know if it is a standard or not, but we've usually been wrapping links around the `<a` and `target="_blank` spots:
`See <a\
 href="https://www.freebsd.org/cgi/man.cgi?query=virtio&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
 target="_blank">VIRTIO(4)</a> for more details.`

This seems to balance the readability and line length pretty well. It also makes it easier to select and replace links.

Comment (Mrt134) [reply src/app/pages/jails/jail-wizard/jail-wizard.component.ts]: Sorry, github is trying to help with the formatting - message me on Rocket and I can get you an example.

File: src/app/pages/jails/jail-wizard/jail-wizard.component.ts [hunk 2]
```diff
@@ -73,11 +77,7 @@ export class JailWizardComponent {
           type: 'input',
           name: 'ip4_addr',
           placeholder: T('IPv4 Address'),
-          tooltip: T('Type the IPv4 address for VNET and shared IP jails.\
- Single interface format: <b>interface|ip-address/netmask</b>. Multiple\
- interface format:\
- <b>interface|ip-address/netmask,interface|ip-address/netmask</b>.\
- Example: <b>vnet0|192.168.0.10/24</b>'),
+          tooltip: T('The IPv4 address for the jail to use.'),
```
Comment (Mrt134) [inline src/app/pages/jails/jail-wizard/jail-wizard.component.ts]: What about "Enter an IPv4 address for the jail." ? Seems to flow a little better.

File: src/app/pages/jails/jail-wizard/jail-wizard.component.ts [hunk 3]
```diff
@@ -106,20 +108,18 @@ export class JailWizardComponent {
           type: 'input',
           name: 'ip6_addr',
           placeholder: T('IPv6 Address'),
-          tooltip: T('Type the IPv6 address for VNET and shared IP jails.\
- Single interface format: <i>interface|ip-address/netmask</i>. Multiple\
- interface format:\
- <i>interface|ip-address/netmask,interface|ip-address/netmask</i>.\
- Example: <b>re0|2001:0db8:85a3:0000:0000:8a2e:0370:7334/24</b>'),
+          tooltip: T('The IPv6 address for the jail to use.'),
```
Comment (Mrt134) [inline src/app/pages/jails/jail-wizard/jail-wizard.component.ts]: Same comment as above with 'ip4_addr'.

File: src/app/pages/jails/jail-wizard/jail-wizard.component.ts (hunk 1 above)
Comment (miker) [reply src/app/pages/jails/jail-wizard/jail-wizard.component.ts]: This was 1 of my first tooltip reviews and didn't realize how we were wrapping URLs yet. The rest of my URLs are wrapped properly.

**Comment by miker** [comment]
After jenkins is done, should be good to merge.

**Comment by erinix** [comment]
please merge master to bring in the build fix

**Pull request merged**